src/resolvers/Mutation/removeComment.ts

```diff
@@ -48,38 +52,43 @@ export const removeComment: MutationResolvers["removeComment"] = async (
     );
   }
 
-  // Checks whether currentUser with _id === context.userId is not the creator of comment.
-  if (comment.creator.toString() !== context.userId.toString()) {
+  const isCurrentUserAdminOfOrganization = currentUser.adminFor.some(
+    (organization) =>
+      organization.toString() === comment.postId.organization.toString()
```

Comment:
@EshaanAgg mongoose has inbuilt method of comparing mongodb objectIds. The method exists on every mongoose objectId. The comparision is done like this
`objectId1.equals(objectId2)`. Read more about it, and I'd say use it throughout the codebase whenever a comparision for mongodb objectIds is encountered.

Reply:
Thanks for the advice!